### PR TITLE
Frontend catch up

### PR DESCRIPTION
**Pull request by AquiGorka**
Still not working, but getting there. Let's get this merged and continue from there.

### CODE REVIEW

File: app/src/script.js [hunk 1]
```diff
@@ -8,28 +8,17 @@ const api = new AragonApi()
...
     switch (event.event) {
       case INITIALIZATION_TRIGGER:
-        newState = {
-          eth: await getValue(),
-          tokens: await getTokens(),
-          ratio: await getCurrentRatio()
-        }
-        break
+      case 'addedToPool':
       case 'Increment':
```
Comment (bingen) [inline app/src/script.js:16]: I guess we don't need Increment and Decrement anymore, right?

File: app/src/script.js [hunk 2]
```diff
@@ -45,13 +34,13 @@ api.store(
 )
 
 async function getValue() {
-  return parseInt(await api.call('value').toPromise(), 10)
+  return api.call('ethValue').toPromise()
```
Comment (bingen) [inline app/src/script.js:37]: Don't we need `parseInt`?

**Review by bingen — APPROVED** [review]

File: app/src/script.js (hunk 1 above)
Comment (AquiGorka) [reply app/src/script.js:16]: True, and I think I misspelled `AddedToPool` should be capitalized.

File: app/src/script.js (hunk 2 above)
Comment (AquiGorka) [reply app/src/script.js:37]: Depends, if we want the values as integers or not, I removed it for the time being to figure out what sort of values we'll be working with here.

**Pull request merged**